config/kubernetes/sig-docs/teams.yaml

```diff
@@ -261,6 +231,25 @@ teams:
     - xichengliudui
     - zhangxiaoyu-zidif
     privacy: closed
+  sig-docs-vi-owners:
+    description: Admins for Vietnamese content
+    members:
+    - ngtuna
+    - truongnh1992
+    privacy: closed
+  sig-docs-vi-reviews:
+    description: PR reviews for Vietnamese content
+    members:
+    - ngtuna
+    - truongnh1992
+    privacy: closed
+  website-admins:
```

Comment:
Can you add a `previously` field here so that the current `kubernetes-website-admins` team is updated instead of creating a new team?

https://github.com/kubernetes/community/blob/master/github-management/org-owners-guide.md#structure